transcendence-app/src/user/dto/user.response.dto.ts

```diff
@@ -1,10 +1,14 @@
 import { User } from '../infrastructure/db/user.entity';
 
+export class UserBlockRelation {
+  isUserBlockedByMe!: boolean;
+  amIBlockedByUser!: boolean;
+}
```

Comment:
since if a user has blocked me is also meaningful, we should get this state in both directions:
a) the user is blocked by me
b) the user has blocked me